source/Generic/PlayState/PlayState.csproj

```diff
@@ -31,9 +33,6 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Playnite.SDK, Version=6.2.2.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>packages\PlayniteSDK.6.2.2\lib\net462\Playnite.SDK.dll</HintPath>
-    </Reference>
```

Comment:
Why was this removed?

Reply:
Was automatically removed by Visual Studio when changing to package reference: https://docs.microsoft.com/en-us/nuget/consume-packages/migrate-packages-config-to-package-reference

This change was necessary because it didn't compile after adding the notifications dependencies without doing so.